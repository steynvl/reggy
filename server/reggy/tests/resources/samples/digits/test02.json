{
  "sampleStringsInfo": [
    [
      {
        "markerType": "Digits",
        "markedStrings": [
          "adsad"
        ],
        "markerInfo": {
          "zero": true,
          "one": true,
          "two": true,
          "three": true,
          "four": false,
          "five": true,
          "six": true,
          "seven": true,
          "eight": true,
          "nine": true,
          "minus": {
            "minus": true,
            "optional": false
          }
        },
        "repeatInfo": {
          "repeat": "1"
        }
      }
    ]
  ],
  "generalRegexInfo": {
    "startRegexMatchAt": "Anywhere",
    "endRegexMatchAt": "Anywhere",
    "regexTarget": "Java"
  },
  "generateMethod": "samplesAndSemantics"
}
