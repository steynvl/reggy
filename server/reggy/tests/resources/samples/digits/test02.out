Java
Pattern regex = Pattern.compile("-[0-35-9]");
-[0-35-9]
Perl
my $regex = /-[0-35-9]/;
-[0-35-9]
POSIX
-[0-35-9]
-[0-35-9]
Python
regex = re.compile(r'-[0-35-9]')
-[0-35-9]
JavaScript
const regex = /-[0-35-9]/;
-[0-35-9]
PHP
$regex = '/-[0-35-9]/';
-[0-35-9]
Golang
regex, _ := regexp.Compile("-[0-35-9]")
-[0-35-9]
Rust
let re = Regex::new(r"-[0-35-9]").unwrap();
-[0-35-9]
C#
var re = new Regex(@"-[0-35-9]");
-[0-35-9]
Scala
val re = "-[0-35-9]".r
-[0-35-9]
Kotlin
val regex = Regex("-[0-35-9]")
-[0-35-9]